docs/integrators/deep-history-squad.md

```diff
@@ -203,15 +199,28 @@ mkdir -p ./db
 Downloading the archives and extracting them might take a while.
 :::
 
-Now, the reconstruction workspace should look as follows (irrelevant files omitted):
+The reconstruction workspace should look as follows (irrelevant files omitted):
```

Comment:
Can add a few extra words, for context / for the reader: `When reconstructing the whole history, the workspace ...`.

Reply:
Done.